### PR TITLE
Connencting filterfunction with badges/ adding a GlobalStyle

### PR DESCRIPTION
**Pull request by oarzani**

### CODE REVIEW

File: src/Pages/Results.js [hunk 1]
```diff
@@ -26,44 +26,73 @@ const FilterSection = styled.section`
...
 export default function Results() {
-  const [filteredStudios, setFilteredStudios] = React.useState(studios);
+  // Suche:
+  // const [search, setSearch] = useState("");
+  // const FilteredByCity = studios.filter(studio =>
+  //   studio.city.toLowerCase().includes(search.toLowerCase())
+  // );
+  // function handleSearchCity(value) {
+  //   setSearch(value);
+  // }
```
Comment (marwinburesch) [inline src/Pages/Results.js]: Please leave comments like this one out, or mark the PR as [WIP]

File: src/api/Studios.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ export const studios = [
     imageLogo: "src",
     imgDetail: "src",
     name: "Fittness First Eherenfeld",
-    price: ["40"],
+    price: [40],
     duration: ["24"],
```
Comment (marwinburesch) [inline src/api/Studios.js]: I don't think these numbers need to be in an array, right?

File: src/Pages/Results.js [hunk 3]
```diff
@@ -26,44 +26,73 @@ const FilterSection = styled.section`
...
       <Header />
 
       <Main>
         <FilterSection>
-          <SecondSearchbar />
+          <SecondSearchbar
+          // onChange={event => handleInputChange(event.target.value)}
+          // handleInputChange={setSearch}
+          // onSearch={handleSearchCity}
+          />
```
Comment (jbaltres) [inline src/Pages/Results.js]: is line 62 and line 85 to 89 not the same?

File: src/api/Studios.js [hunk 4]
```diff
@@ -252,3 +255,46 @@ export const studios = [
     }
   }
 ];
+
+// Ein Versuch, aus Around-Spaces adaptiert src/api/restaurants.js
```
Comment (jbaltres) [inline src/api/Studios.js]: you should delete this one

Comment (oarzani) [reply src/api/Studios.js]: thanks

**Pull request merged**